Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -10922,6 +10926,89 @@ a_Player:OpenWindow(Window);
 			},
 			Inherits = "cEntity",
 		},
+		cExpOrb =
+		{
+			Desc = [[
+				This class represents a experience orb. This entity can be spawned by using cWorld:SpawnExperienceOrb.
+				It's also spawned when a monster is killed or a player is mining coal.
+			]],
+			Functions =
+			{
+				GetAge =
```

Comment:
What are the Age methods used for? I find it strange that the age of an entity can be changed. Also, any idea why only the experience orb has an age value?